app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationListViewModel.kt

```diff
@@ -35,26 +44,29 @@ class ConversationListViewModel @Inject constructor(
         }
     }
 
-    private val _state = MutableStateFlow(ConversationListState())
-
-    val state: StateFlow<ConversationListState>
-        get() = _state
+    var state by mutableStateOf(ConversationListState())
+        private set
 
     init {
-        _state.value = ConversationListState(
-            newActivities = newActivitiesMockData,
-            conversations = conversationMockData,
-            missedCalls = mockMissedCalls,
-            callHistory = mockCallHistory,
-            unreadMentions = mockUnreadMentionList,
-            allMentions = mockAllMentionList,
-            unreadMentionsCount = 12,
-            missedCallsCount = 100,
-            newActivityCount = 1
-        )
+        viewModelScope.launch {
+            getConversationUseCase.invoke()
```

Comment:
Same comment as above, maybe we can call this just `getConversations()` for simplicity